src/Apps/Artwork/Components/TrustSignals/SecurePayment.tsx

```diff
@@ -20,7 +23,10 @@ export const SecurePayment = ({ artwork, ...other }: SecurePaymentProps) => {
           <>
             {"Secure transactions by credit card through Stripe."}
             <br />
-            <Link href="https://stripe.com/docs/security/stripe">
+            <Link
+              href="https://stripe.com/docs/security/stripe"
+              target="_blank"
```

Comment:
Thank you!

Reply:
```suggestion
              target="_blank"
              rel=”noopener noreferrer”
```

Out of an abundance of caution, it's always good to add this `rel` to `target="_blank"` links, unless we specifically want to allow them to have some limited ability to control our site. 

Read more here: https://developers.google.com/web/tools/lighthouse/audits/noopener. We should probably bake this into the link as a prop (like `newTab` or something).

Reply:
🙀 `window.opener.location = newURL`